other/edge-server/client/node-server/README.md

```diff
@@ -0,0 +1,2 @@
+TODO: Write info about this server and how it fits into the Edge Server template
```

Comment:
Oops. We should probably either populate details here or remove this README.md from the branch.